test/test_helper.cpp

```diff
@@ -108,3 +109,178 @@ TEST(Functionality, Subsets) {
                         std::pair<size_t, size_t>(3, 4)) !=
               subsets_of_size_2.end());
 }
+
+TEST(VSSModel, Consistency) {
```

Comment:
I am guilty of this myself so no shade here. But generally it is better to have small unittests that test one specific thing. That way, if something fails halfway through the test you can still get information for the rest of the tests without needing to incrementally fix and re-run. This often leads to a lot of boilerplate code though.

Reply:
Thanks for the comment. I would leave it for now since they pass, but make sure to have small(er) test cases for future enhancements.